app/test_main.py

```diff
@@ -1,3 +1,73 @@
+from typing import Any
+
+import pytest
+
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
```

Comment:
The test case `(14, 14, [0, 0])` might be incorrect if the logic of `get_human_age` considers 14 animal years to be equivalent to some human years. Verify the logic in `get_human_age` to ensure this test case is valid.